lib/stack_master/config.rb

```diff
@@ -74,7 +74,7 @@ def load_stacks(stacks)
         region = Utils.underscore_to_hyphen(region)
         stacks_for_region.each do |stack_name, attributes|
           stack_name = Utils.underscore_to_hyphen(stack_name)
-          stack_attributes = build_stack_defaults(region).deeper_merge(attributes).merge(
+          stack_attributes = build_stack_defaults(region).deeper_merge!(attributes).merge(
```

Comment:
I'm not sure I understand. Are you saying that the default s3 config couldn't be overridden on a per-stack basis or that the override bled into other stack definitions?